src/applications/proxy-rewrite/tests/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  chrome:
+    build: ./chrome
+    shm_size: '1gb'
+    cap_add:
+      - SYS_ADMIN
+    ports:
+      - '9222'
+  vrt:
+    build: ./vrt
+    volumes:
+      - .:/application
+    links:
```

Comment:
`links` is deprecated but I'm not really sure how to set up a network between containers or if it's necessary.